### PR TITLE
`include` setting on item

### PR DESCRIPTION
**Pull request by tg666**
- added setting `include` to item's configuration (value can be string or array of strings)
- added method `IMenuItem::setInclude()`
- added test

### CODE REVIEW

**Comment by davidkudera** [comment]
Thank you, good work :+1: 

Do you think you could also add some note into the readme file please?

**Comment by tg666** [comment]
Of course, here it is :slightly_smiling_face:

**Comment by tg666** [comment]
@davidkudera can you merge it please? :slightly_smiling_face:

**Comment by davidkudera** [comment]
@tg666 sure, I was just waiting for you with the review. You should see one a bit above here. Can you please fix it? I'll merge it immediately afterwards.

File: src/Carrooi/Menu/Loaders/ArrayMenuLoader.php [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 namespace Carrooi\Menu\Loaders;
 
+use Nette\Utils;
```
Comment (jelen07) [inline src/Carrooi/Menu/Loaders/ArrayMenuLoader.php]: Sort it alphabetically please.

Comment (tg666) [reply src/Carrooi/Menu/Loaders/ArrayMenuLoader.php]: Done

**Comment by davidkudera** [comment]
Thank you and good job :+1: 

also sorry for the problems....

**Comment by tg666** [comment]
No problem, thank you too :+1:

**Pull request merged**